packages/react-microlink/src/components/Card/CardContent.js

```diff
@@ -15,36 +16,46 @@ export const Content = styled.div`
 const Title = styled.h2`
   font-size: 16px;
   margin: 0 0 8px;
-  white-space: nowrap;
```

Comment:
Did you remove this from _desktop_ on purpose @Kikobeats? Without that, the title can span _infinite_ lines depending on its content length.

It makes the load a bit less smooth I think:
![microlink_card_load](https://user-images.githubusercontent.com/5795227/34906378-c2fdfaa0-f86c-11e7-9249-21aeab4e1e57.gif)

Reply:
Good catch, added again.